src/main/java/org/example/repository/ApartmentRepository.java

```diff
@@ -0,0 +1,16 @@
+package org.example.repository;
+
+import org.example.entity.Apartment;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface ApartmentRepository extends JpaRepository<Apartment, Integer> {
+
+    List<Apartment> findAllByBuilding_Id(int id);
+
+    List<Apartment> findAllByRealtyAgent_Id(int id);
+
+    Apartment findById(int id);
```

Comment:
No)
I mean, that this method already present in JpaRepository.
Let's discuss on the lesson